feat: IPFS retrieval client (#243)
filecoin-station/zinnia: docs/building-modules.md

Call this function every time your module completes a job. It's ok to call it frequently.

### IPFS Retrieval Client

Zinnia provides a built-in IPFS retrieval client making it easy to fetch content-addressed data from

> IPFS and Filecoin networks. You can retrieve data for a given CID using the web platform API `fetch`
> and using the URL scheme `ipfs://`.

Reviewer: ```suggestion
IPFS and Filecoin networks. You can retrieve data for a given CID using the web platform API `fetch`
together with the URL scheme `ipfs://`.
```

Example:

```js
const response = await fetch("ipfs://bafybeib36krhffuh3cupjml4re2wfxldredkir5wti3dttulyemre7xkni");
assert(response.ok);
const data = await response.arrayBuffer();
// data contains binary data in the CAR format
```

> > Note: At the moment, Zinnia does not provide any tools to interpret the returned CAR data. We are

Reviewer: ```suggestion
> Note: At the moment, Zinnia does not provide any tools for interpreting the returned CAR data. We are
```

> discussing support for reading UnixFS data in
> [zinnia#245](https://github.com/filecoin-station/zinnia/issues/246).

> Under the hood, Zinnia handles `ipfs://bafy...` requests by calling Lassie HTTP API. You can learn

Reviewer: ```suggestion
Under the hood, Zinnia handles `ipfs://bafy...` requests by calling Lassie's HTTP API. You can learn
```
